extensions/src/main/java/org/pentaho/platform/web/http/filters/PentahoWebContextFilter.java

```diff
@@ -44,20 +44,22 @@
 
 import java.io.IOException;
 import java.io.OutputStream;
+import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Locale;
 import java.util.Map;
 
 /**
  * If the request is searching for a webcontext.js, it writes out the content of the webcontext.js
- *
- * @author Ramaiz Mansoor
- *
  */
 public class PentahoWebContextFilter implements Filter {
 
   public static final String WEB_CONTEXT_JS = "webcontext.js"; //$NON-NLS-1$
+
+  private static final String REQUIREJS_LOCATION = "content/common-ui/resources/web/require.js";
+  private static final String REQUIREJS_CONFIG_LOCATION = "content/common-ui/resources/web/require-cfg.js";
```

Comment:
The pentaho platform project itself could provide the require.js and initializer file...
Or, maybe when we split common-ui?